libs/js/banglejs/Bangle_setUI_Q3.js

```diff
@@ -123,10 +123,12 @@
       };
       Bangle.on("touch", Bangle.touchHandler);
     }
-    var btnWatch = setWatch(function() {
-      btnWatch = undefined;
-      options.back();
-    }, BTN1, {edge:"falling"});
+    if (!options.btn) { // Back function does not override if other function has been assigned to btn
+      var btnWatch = setWatch(function() {
```

Comment:
I think while this works, the scoping of `btnWatch` is wrong because it's used in `remove` on line 136. On Espruino, if options.btn and back were defined, then the widget were removed, there's be a ReferenceError.